app/models/event_type.rb

```diff
@@ -36,6 +36,7 @@ class EventType < ActiveRecord::Base
   # Associations ###############################################################
 
   has_many :events
+  has_one  :position
```

Comment:
Style/SingleSpaceBeforeFirstArg: Put one space between the method name and the first argument.